src/components/molecules/OcBreadcrumb/OcBreadcrumb.vue

```diff
@@ -135,9 +141,15 @@ export default {
     > li a,
     > li button,
     > li span,
-    > :nth-child(n + 2):not(.uk-first-column)::before {
+    > :nth-child(n + 2)::before {
       color: var(--oc-color-text-muted);
     }
+    > :nth-child(n + 2)::before {
+      content: "/";
+      display: inline-block;
+      margin: 0 20px 0 calc(20px - 4px);
```

Comment:
Can you please reduce the margins by at least 50%?

Reply:
Sure thing, done!